crates/ruff_linter/src/rules/airflow/snapshots/ruff_linter__rules__airflow__tests__AIR303_AIR303.py.snap

```diff
@@ -0,0 +1,74 @@
+---
+source: crates/ruff_linter/src/rules/airflow/mod.rs
+snapshot_kind: text
+---
+AIR303.py:10:1: AIR303 Import path `airflow.api.auth.backend.basic_auth` is moved into `fab` provider in Airflow 3.0;
+                        Install `apache-airflow-provider-fab==1.0.0` and import from `airflow.providers.fab.auth_manager.api.auth.backend.basic_auth` instead.
+   |
+ 8 | from airflow.www.security import FabAirflowSecurityManagerOverride
+ 9 | 
+10 | basic_auth, kerberos_auth
+   | ^^^^^^^^^^ AIR303
+11 | auth_current_user
+12 | backend_kerberos_auth
+   |
```

Comment:
I think this is a pretty long message, it might be useful to break it down into two parts - the description and the suggestion. For the description we can avoid the entire import path and just provide the symbol name itself while for the help text we can guide the user to install the provider and use the provider namespace for the symbol. For example:

```
AIR303.py:10:1: AIR303 `basic_auth` will be moved into `fab` provider in Airflow 3.0
   |
 8 | from airflow.www.security import FabAirflowSecurityManagerOverride
 9 | 
10 | basic_auth, kerberos_auth
   | ^^^^^^^^^^ AIR303
11 | auth_current_user
12 | backend_kerberos_auth
   |
   = help: Install `apache-airflow-provider-fab` and import from `airflow.providers.fab` instead
```

The help text is rendered using the `fix_title` method: https://github.com/astral-sh/ruff/blob/a3bb0cd5ec4d4dffea5d42d184e8c4a062976663/crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs#L65-L72

I'm also a bit worried about the version number for the package, I'd prefer to omit them as it can get outdated pretty quickly.

Curious to know your thoughts on this :)

Reply:
> I'm also a bit worried about the version number for the package, I'd prefer to omit them as it can get outdated pretty quickly.

Agree. What's the motivation for including the version? Is it because they need a specific minimal version?

Reply:
> Agree. What's the motivation for including the version? Is it because they need a specific minimal version?

Yep, some module moves might not happen in the `1.0.0` version of that provider.

Reply:
We could potentially use `>=1.0.0,<2` presuming that the changes follow semantic versioning. The `1.0.0` can be updated to the provider version in which the change takes place.

Reply:
sounds good! this is a great suggestion!

Reply:
I'm not sure about the upper bound. Maybe just mention *newer than x.x.x*

Reply:
After a second thought, yep, it would make things easier, `>=1.0.0` sounds like a good enough solution 🤔

Reply:
Updated!